config/locales/de_DE.yml

```diff
@@ -578,6 +578,7 @@ de_DE:
     shared_access_success: Raum erfolgreich geteilt
     shared_access_error: Beim Teilen des Raums ist ein Fehler aufgetreten
     start: Starten
+    search: Nach Raum suchen…
```

Comment:
This auto-generated by Transifex service.

Reply:
That's true. I assumed that the languages will always be overwritten so that modifications like this won't be a problem while this might help for testing if translations actually work. I can remove this if you want me to?

Reply:
Yeah - please remove it. Makes merging translations late easier